pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "epigraphhub"
-version = "1.2.0"  # changed by semantic-release
+version = "1.1.1"  # changed by semantic-release
```

Comment:
I think it shouldn't be changed here

Reply:
I guess that all the changes on pyproject is not correct here .. probably it is from an old version.